src/App.tsx

```diff
@@ -35,3 +33,24 @@ export function App({
     </HelmetProvider>
   )
 }
+
+const SkipToContent = connect(function SkipToContent() {
+  return (
+    <a href="#main" className="btn skip-to-content">
+      {localizeSkipToContent()}
+    </a>
+  )
+})
+
+function localizeSkipToContent(): string {
+  switch (currentLanguage()) {
+    case 'de':
+      return 'Zum Inhalt springen'
+    case 'fr':
+      return 'Aller au contenu'
+    case 'pl':
+      return 'Przejdź do treści'
+    default:
+      return 'Skip to Content'
```

Comment:
* [ ] FYI Not sure which would be the most consistent, but I quickly found `localizeLogOutLabel` which uses sentence case. I actually expected sentence case here, not title case.

Reply:
Nice catch!